nuxeo-ai-core/src/main/java/org/nuxeo/ai/configuration/ThresholdConfiguratorDescriptor.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * (C) Copyright 2006-2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Contributors:
+ *     anechaev
+ */
+package org.nuxeo.ai.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.nuxeo.common.xmap.annotation.XNode;
+import org.nuxeo.common.xmap.annotation.XNodeList;
+import org.nuxeo.common.xmap.annotation.XObject;
+
+@XObject("thresholdConfiguration")
+public class ThresholdConfiguratorDescriptor {
```

Comment:
You should `implements Descriptor`

Reply:
this is not true. Unless you want your descriptor to be a part of `DefaultComponent`. I'm taking care of registering it myself, so no point to have the interface.